src/Containers/SlidingOutPanels.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { MainInterface, ResponsiveInterface } from '../Utils/BaseStyles';
+
+interface ImageProps {
+    id: string;
+    imageSource: string;
+    alt: string;
+}
+
+export interface SlidingOutPanelProps
+    extends MainInterface,
+        ResponsiveInterface {
+    images: ImageProps[];
+}
+
+export const SlidingOutPanels: React.FC<SlidingOutPanelProps> = ({
+    images,
+}): React.ReactElement => {
+    return (
+        <Items>
+            <Overlay>
+                {images.map(
+                    ({ id, imageSource, alt }): React.ReactElement => (
+                        <Image key={id} id={id} src={imageSource} alt={alt} />
```

Comment:
Extensibility is important, What happens if someone wants to click on each panel to go to a different link per Image?

Reply:
@aslamm29 Once you solve this design question, you can merge away!

Reply:
@ralph-dev would enclosing it with a anchor tag fit this purpose or do you recommend an onclick handler to make it entirely customizable?

Reply:
I would suggest an onClick tag becuase from Extensibility, you can cover both scenarios without concern. But from UX you want to let the user navigate to where they clicked essentially based on the image.

Reply:
i have added the onClick handler to the image component